test/tests.zig

```diff
@@ -1279,6 +1283,29 @@ pub fn addCAbiTests(b: *std.Build, skip_non_native: bool, skip_release: bool) *S
     return step;
 }
 
+pub fn addTranslateCases(
+    b: *std.Build,
+    parent_step: *Step,
+    opt_test_filter: ?[]const u8,
+    translate_options: @import("src/Cases.zig").TranslateOptions,
+) !void {
+    const arena = b.allocator;
+    const gpa = b.allocator;
+
+    var translate_cases = @import("src/Cases.zig").init(gpa, arena);
+
+    var dir = try b.build_root.handle.openDir("test/cases", .{ .iterate = true });
+    defer dir.close();
+
+    translate_cases.addFromDir(dir, b);
```

Comment:
This is going to make `zig build` take longer since it will have to find and parse all the cases an additional time. Instead, this work needs to continue to be done only once.

Reply:
Good point, I'll change that.